addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -157,19 +157,19 @@
             <t t-if="reconciled_moves">
                 <DatiFattureCollegate t-foreach="reconciled_moves" t-as="reconciled_move">
                     <IdDocumento t-out="format_alphanumeric(reconciled_move.name, -20)"/>
-                    <DataDocumento t-out="format_date(reconciled_move.date if reconciled_move.l10n_it_edi_is_self_invoice else reconciled_move.invoice_date)"/>
+                    <Data t-out="format_date(reconciled_move.date if reconciled_move.l10n_it_edi_is_self_invoice else reconciled_move.invoice_date)"/>
```

Comment:
This is awkward https://github.com/odoo/odoo/commit/98cc8b702cbb38ba1f9c990930d0b0abf3c72182 😬 
@mma1377 @Megaaaaaa Is there something more to it?